app/src/main/res/layout/fragment_thread_list.xml

```diff
@@ -15,275 +15,289 @@
   ~ You should have received a copy of the GNU General Public License
   ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
   -->
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.slidingpanelayout.widget.SlidingPaneLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/threadListSlidingPaneLayout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context=".ui.main.folder.ThreadListFragment">
 
-    <com.google.android.material.appbar.AppBarLayout
-        android:id="@+id/appBarLayout"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent">
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="600dp"
```

Comment:
Do we really want to only start showing the two panes above 600 + 550 dp and not before? Also isn't 600dp a bit big for some smaller tablette widths?

Reply:
Also this should be in a dimension if it is shared with another layout

Reply:
Why 600dp ? Because of this : https://developer.android.com/guide/topics/large-screens/support-different-screen-sizes

The 500dp is a little more arbitrary. If we were totally following the previously linked documentation, we should put 240dp instead of 500dp, but on some tablets it's not enough (for exemple, on Pixel C).

With some trial and error, I found 500dp to be nice. The value itself is not really important, it just needs to be big enough to not show up when the tablet is in portrait, but while still being inferior to the tablet width.

Reply:
We still need to block TwoPane layout in portrait mode.
Right now, without handling configuration changes, it's impossible.
For now we block it until screens of 1100 width.